version/1/3/0/de/code_of_conduct.md

```diff
@@ -0,0 +1,24 @@
+# Verhaltenskodex für Mitwirkende
+
+Als die Mitwirkenden und die Verantwortlichen dieses Projekts, und in dem Willen, eine offene und einladende Gemeinschaft zu fördern, geloben wir, alle Personen zu respektieren, die zum Projekt duch das Anlegen von Support-Tickets, dem Veröffentlichen von Feature Requests, dem Überarbeiten von Dokumentation, dem Vorschlagen von Pull Requests oder Patches und durch andere Aktivitäten beitragen.
+
+Wir verpflichten uns, die Mitwirkung an diesem Projekt zu einer belästigungsfreien Erfahrung für alle zu machen, unabhängig von Kenntnisstand, Geschlecht, Geschlechtsidentität und -Ausdruck, sexueller Orientierung, Behinderung, äußerlicher Erscheinung, Körpermaßen, Rasse, Ethnizität, Alter, Religion oder Nationalität.
+
+Beispiele für nicht akzeptables Verhalten beinhalten:
+
+* Die Verwendung sexualisierter Sprache oder Symbolik
```

Comment:
I'm a little bit unsure about translating "imagery" to "Symbolik". Google tells me the definition of imagery is:

1. visually descriptive or figurative language, especially in a literary work. "Tennyson uses imagery to create a lyrical emotion", visual symbolism. "the film's religious imagery"
2. visual images collectively. "the impact of computer-generated imagery on contemporary art"

"Symbolik" corresponds to meaning 1. @CoralineAda could you clarify which of the two versions you mean more? If it's meaning 2, maybe simply "Bilder" (pictures) instead?